### ISSUE TITLE
Stick to similar words in code and design

### ISSUE THREAD

**Issue by mtbitcr** [BitcreditProtocol/Bitcredit-Core#213]
We have an issue in our code that sometimes the terms in design, frontend and backend are different. This needs to be fixed. The correct terminology is the one used in design.

**Comment by HubertusVIE** [comment]
I have not gone too much into correcting text during design. Only those where it is  legally imperative. Will I have a possibility to change text on the fly and update the app?Sent from my iPhoneOn 11. Nov 2024, at 09:51, MT ***@***.***> wrote:﻿
We have an issue in our code that sometimes the terms in design, frontend and backend are different. This needs to be fixed. The correct terminology is the one used in design.

**Comment by mtbitcr** [comment]
> Will I have a possibility to change text on the fly and update the app?

Only with App updates.

**Comment by HubertusVIE** [comment]
Of course, I meant will I be able to play around with text and see it in a simulation for instance?Sent from my iPhoneOn 12. Nov 2024, at 11:41, MT ***@***.***> wrote:﻿

Will I have a possibility to change text on the fly and update the app?

Only with App updates.
